Assets/MixedRealityToolkit.Tests/PlayModeTests/Components/FocusRaycastTestProxy.cs

```diff
@@ -29,7 +29,7 @@ public class FocusRaycastTestProxy : MonoBehaviour
 
         /// <summary>
         /// The number of RaySteps to generate from <see cref="RayLineData"/>.
-        /// Corresponds to <see cref="Toolkit.Input.LinePointer.LineCastResolution"/> and is used to set that value on the <see cref="TestPointer"/>.
+        /// Corresponds to <see cref="Toolkit.Input.CurvePointer.LineCastResolution"/> and is used to set that value on the <see cref="TestPointer"/>.
```

Comment:
Do we need any new tests for the curve pointer and line pointer? Perhaps things that test that a curve pointer actually collides against colliders along the curve, and line pointers collider along colliders that are straight. Just realized we may not have tests for this.

Reply:
hmmm, curvepointer can be tested through the teleportpointer implementation...but not sure what tests we have there. 

The main test I should look into (if it already exists) for LinePointer is the IsFocusLocked code (which was pre-existing logic that was preserved). We set different start/end points based on that if condition which would be good to validate in a test (again if it doesn't already exist).

Reply:
Test added